### PR TITLE
0.0.2 csv

### PR DESCRIPTION
**Pull request by rthallisey**
Set 0.0.1 CSV to have the release-2.0 tags.  0.0.2 will point to release-2.1 when the branch is created.

### CODE REVIEW

**Comment by rthallisey** [comment]
@rwsu this should give you the automation to build the new CSV to upgrade to

File: deploy/converged/olm-catalog/kubevirt-hyperconverged/0.0.2/cdi.crd.yaml [hunk 1]
```diff
@@ -0,0 +1,32 @@
+---
```
Comment (djzager) [inline deploy/converged/olm-catalog/kubevirt-hyperconverged/0.0.2/cdi.crd.yaml]: The path should now be `deploy/olm-catalog/kubevirt-hyperconverged/0.0.2/...`, maybe you need to rebase and rebuild the manifests?

Comment (rthallisey) [reply deploy/converged/olm-catalog/kubevirt-hyperconverged/0.0.2/cdi.crd.yaml]: I think this is leftovers..

Comment (djzager) [reply deploy/converged/olm-catalog/kubevirt-hyperconverged/0.0.2/cdi.crd.yaml]: Looks like you have both. Maybe just remove `deploy/converged`.

Comment (rthallisey) [reply deploy/converged/olm-catalog/kubevirt-hyperconverged/0.0.2/cdi.crd.yaml]: I thought the rebase was too easy :).  Removed.

**Comment by rthallisey** [comment]
CI failure captured in https://github.com/kubevirt/cluster-network-addons-operator/issues/189.  Not related to the HCO code.

**Pull request merged**